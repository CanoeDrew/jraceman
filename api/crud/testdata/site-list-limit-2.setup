INSERT into site(id, name, city, phone) values
("S4", "Site Four", "Middleburg", "800-555-1111"),
("S5", "Site Five", "Tulies", "888-555-2222");
