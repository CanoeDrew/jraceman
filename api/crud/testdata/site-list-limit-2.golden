[
  {
    "ID": "S3",
    "Name": "Site Three",
    "Street": null,
    "Street2": null,
    "City": "Nowhere",
    "State": null,
    "Zip": null,
    "Country": null,
    "Phone": "888-555-9999",
    "Fax": null
  },
  {
    "ID": "S4",
    "Name": "Site Four",
    "Street": null,
    "Street2": null,
    "City": "Middleburg",
    "State": null,
    "Zip": null,
    "Country": null,
    "Phone": "800-555-1111",
    "Fax": null
  }
]